### ISSUE TITLE
Default install for pip does not yet work on Python 3.7.3 (but works on Python 3.6.2)

### ISSUE THREAD

**Issue by sharpe5** [ssanderson/interface#28]
If I run `pip install python-interface` on Python 3.7.3, I get this error:

```
Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)] on win32
from interface import implements, Interface
Traceback (most recent call last):
  File "S:\bin\Python37\lib\site-packages\IPython\core\interactiveshell.py", line 3296, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-ba8f47d4b56b>", line 1, in <module>
    from interface import implements, Interface
  File "C:\Program Files\JetBrains\PyCharm 2018.3.2\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
ModuleNotFoundError: No module named 'interface'
```

Everything works perfectly for Python v3.6.2.

Perhaps the latest version from Git should be pushed to PyPi?

**Comment by sharpe5** [comment]
Update: I installed from GitHub, and everything worked fine on Python 3.7.3.

Then I used this to reinstall the old version from pip:

```
pip install --upgrade --force-reinstall python-interface
```

And, weirdly enough, everything was still working.

So I'm not sure why it gives the error above on a fresh install of Python 3.7.3, but installing from source, then reinstalling the same package using pip seems to fix it?
